src/main/java/ui/components/pickers/PickerRepository.java

```diff
@@ -0,0 +1,75 @@
+package ui.components.pickers;
+
+import javafx.geometry.Insets;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Border;
+
+/**
+ * This class is used to represent a repo in RepositoryPicker.
+ *
```

Comment:
The class's header comment can focus more on what it does than how it's used. So this can be `This class represents a repository...`. Same for other classes' header comments.